npm/podController_test.go

```diff
@@ -211,7 +214,25 @@ func TestAddMultiplePods(t *testing.T) {
 	podObj1 := createPod("test-pod-1", "test-namespace", "0", "1.2.3.4", labels, NonHostNetwork, corev1.PodRunning)
 	podObj2 := createPod("test-pod-2", "test-namespace", "0", "1.2.3.5", labels, NonHostNetwork, corev1.PodRunning)
 
-	f := newFixture(t)
+	var calls = []testutils.TestCmd{
```

Comment:
nitpick: better to use string variables for them?

Reply:
Wondering what the best practice is in UTs? It seems many elements in TestCmd are duplicated in multiple places. Can we remove the redundancy?

Reply:
discussed offline, will add helper methods in subsequent pr